Resources/config/form-type.xml

```diff
@@ -9,5 +9,13 @@
             <tag name="form.type" alias="cmf_core_checkbox_url_label" />
             <argument type="service" id="router" />
         </service>
+
+        <service id="cmf_core.form.extension.publishable" abstract="true" class="Symfony\Cmf\Bundle\CoreBundle\Form\Extension\PublishableExtension">
+            <argument></argument><!-- Must be overrided when using this service as parent -->
```

Comment:
> this makes me think that we could do a compiler pass for a tag on form types that we tag with form_extension and a parameter that says e.g. publishable. the pass could then copy this service and set the parameter to the tagged service. does nothing in that direction already exist? anyways, this was just a brainstorm, we can also do that in a separate pull request if it makes sense at all.

Since form types are used though their fqcn, there are no longer registered as service. Doing so will require services to be defined. The other solution is to propose a change in `Symfony\Component\Form\Extension\DependencyInjection\DependencyInjectionExtension` to allow `FormTypeExtensions` to extends types implementing some interfaces. This later could be nice, but will require us to bump Symfony dependency really high.